<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BoolParamWidget_UI</class>
 <widget class="QWidget" name="BoolParamWidget_UI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>155</width>
    <height>58</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="verticalSpacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="1">
    <widget class="QCheckBox" name="m_checkBox">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="m_labelName">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QWidget" name="m_widgetComment" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>15</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="m_labelComment">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
